pkg/pipeline/source/sdk.go

```diff
@@ -326,16 +380,21 @@ func (s *SDKSource) onTrackSubscribed(track *webrtc.TrackRemote, pub *lksdk.Remo
 	if !s.initialized.IsBroken() {
 		s.mu.Lock()
 		switch s.RequestType {
+		case types.RequestTypeParticipant:
+			s.filenameReplacements["{publisher_identity}"] = s.Identity
```

Comment:
would we make pID a replacement policy too? not sure if that needs to be onTrackSubscribed tho.

Reply:
could add ID if/when someone asks, I think for the most part identity is more useful

Reply:
yeah you are right.. this is participant composite.. identity is the main thing people would care about.